### ISSUE TITLE
Cut v1.27.13 release

### ISSUE THREAD

**Issue by saschagrunert** [kubernetes/sig-release#2484]
## Release Steps

- [x] Create a thread on #release-management: https://kubernetes.slack.com/archives/CJH2GBF7Y/p1713249381567589
- [x] Screenshot unhealthy release branch testgrid boards
- [x] Check the health of the publishing-bot  
- Mock Run
  - [x] Stage
  - [x] Release
- NoMock Run
  - [x] Stage
  - [x] Image Promotion: https://github.com/kubernetes/k8s.io/pull/6712
  - [x] Release
- [x] Notify #release-management: https://kubernetes.slack.com/archives/CJH2GBF7Y/p1713309696324509
- [ ] Send notification: 
- [ ] Update [`schedule.yaml` file](https://github.com/kubernetes/website/blob/main/data/releases/schedule.yaml) with the latest release 
- [x] Collect metrics and add them to the `Release steps` table below

## Release Tools Version

```
GitVersion:    v0.16.7
GitCommit:     unknown
GitTreeState:  unknown
BuildDate:     unknown
GoVersion:     go1.22.1
Compiler:      gc
Platform:      linux/amd64
```

## Release Jobs History

|      STEP      |                                             COMMAND                                             |                                                     LINK                                                      |             START              | DURATION | SUCCEEDED? |
|----------------|-------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------|--------------------------------|----------|------------|
| `mock stage`   | `krel stage --type official --branch release-1.27 --build-version v1.27.12-18+7c861b1ecad97e`   | https://console.cloud.google.com/cloud-build/builds/ae76a319-77eb-40e4-926c-de0eef88f68c?project=648026197307 | 2024-04-16T06:42:14.701142125Z | 00:56:07 | Yes        |
| `mock release` | `krel release --type official --branch release-1.27 --build-version v1.27.12-18+7c861b1ecad97e` | https://console.cloud.google.com/cloud-build/builds/23cf3789-bc25-4c0b-95d5-6a9512c903f3?project=648026197307 | 2024-04-16T07:39:52.339770417Z | 00:17:48 | Yes        |
| Stage | `krel stage --type official --branch release-1.27 --build-version v1.27.12-18+7c861b1ecad97e --nomock` |https://console.cloud.google.com/cloud-build/builds/d2f157df-b79e-421b-bb41-6d430c7c90a3?project=kubernetes-release-test |April 16, 2024 at 6:49:34 PM UTC+4 |1 hr 1 min |Yes |
| Release | `krel release --type official --branch release-1.27 --build-version v1.27.12-18+7c861b1ecad97e --nomock` |https://console.cloud.google.com/cloud-build/builds/e897194c-38ed-44ef-b56f-d09660af3f21?project=kubernetes-release-test |April 16, 2024 at 11:20:00 PM UTC+4 |16 min 42 sec |Yes |

## Action Items

None

## Open Questions

None

/milestone v1.30
/assign
/cc @kubernetes/release-managers
/priority important-soon
/kind documentation

**Comment by saschagrunert** [comment]
### Testgrid dashboards for 1.27

Boards checked for FAILING:
- [sig-release-1.27-blocking](https://testgrid.k8s.io/sig-release-1.27-blocking)
- [sig-release-1.27-informing](https://testgrid.k8s.io/sig-release-1.27-informing)


<details><summary><tt>2024-04-16 06:25:29.922388691 +0000 UTC</tt> FAILING <a href="https://testgrid.k8s.io/sig-release-1.27-blocking#gce-cos-k8sstable3-alphafeatures&width=30">sig-release-1.27-blocking#gce-cos-k8sstable3-alphafeatures - TestGrid</a></summary><p>

sig-release-1.27-blocking#gce-cos-k8sstable3-alphafeatures

</p></details>




**comment generated by [krel](https://github.com/kubernetes/release/tree/master/docs/krel)**

**Comment by saschagrunert** [comment]
Opened https://github.com/kubernetes/kubernetes/issues/124331 for the failling test. It fails since quite some time and directly on test setup so I'm assuming it's something we can adjust by updating the job. Continuing for now.

**Comment by MaxRink** [comment]
>     vsphere-iso.vsphere: fatal: [default]: FAILED! => {"cache_update_time": 1713350198, "cache_updated": false, "changed": false, "msg": "no available installation candidate for kubelet=1.27.13-1.1"}


looks like the release pipeline for debs failed
